components/apiserver-proxy/README.md

```diff
@@ -0,0 +1,49 @@
+# API Server Proxy
+
+## Overview
+
+The Kyma API Server Proxy is a core component that secure access to the kubernetes apiserver using JWT authentication. 
+It is based on `kube-rbac-proxy`(https://github.com/brancz/kube-rbac-proxy) project.
+This [Helm chart](/resources/core/charts/apiserver-proxy/Chart.yaml) defines the component's installation.
+
+## Prerequisites
+
+You need these tools to work with the API Server Proxy:
+
+- [Go distribution](https://golang.org)
+- [Docker](https://www.docker.com/)
+
+
+## Details
+
+This section describes how to run the controller locally, how to build the Docker image for the production environment, how to use the environment variables, and how to test the Kyma API Server Proxy.
```

Comment:
This section describes:

- How to run the controller locally
- How to build the Docker image for the production environment
- How to use the environment variables
- How to test the Kyma API Server Proxy